changelog/duplicate-implementations-deprecation.dd

```diff
@@ -0,0 +1,26 @@
+Usage of identical functions in a single module
+
+A binary should never contain multiple versions of a symbol.
```

Comment:
It's not that it shouldn't, it simply **cannot**. Also this is a weird sentence to start the longer explanation with.